velox/exec/tests/utils/CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ target_link_libraries(
   velox_type_fbhive
   velox_hive_connector
   velox_tpch_connector
+  velox_tpcds_connector
```

Comment:
We can do this linking only if ${VELOX_ENABLE_TPCDS_CONNECTOR} is set. Seems like we are assuming that here. Can you check ?

Also when including the headers, it should be done only if ${VELOX_ENABLE_TPCDS_CONNECTOR} is set as well.

Reply:
Alright. Thanks.